schedule/security/audit.yaml

```diff
@@ -0,0 +1,7 @@
+name: audit
+description:    >
+    Audit test for Tumbleweed
```

Comment:
We don't need to test this on SLES, correct?

Reply:
Lily said it will later be move to osd, but I just added this case to osd and did verify run on sle, so it's now supporting both tw and sle.